tests/src/Fixtures/TripalImporter/TestTripalImporter.php

```diff
@@ -0,0 +1,606 @@
+        // Data Row Validation
+        // ********************************************************************
+        elseif ($line_no > 1 && isset($validators['data-row'])) {
+          $data_row = ImportValidationHelper::splitRowIntoColumns($line, $file_mime_type);
+
+          foreach ($validators['data-row'] as $validator_name => $validator) {
+            if (!array_key_exists($validator_name, $failures)) {
+              $failures[$validator_name] = [];
+            }
+
+            $result = $validator->validateRow($data_row);
+            if (array_key_exists('valid', $result) && $result['valid'] === FALSE) {
+              $row_has_failed = TRUE;
+              $failed_validator = TRUE;
+              $failures[$validator_name][$line_no] = $result;
+            }
+          }
+        }
+      }
+
+      fclose($handle);
+    }
+
+    $validation_feedback = $this->processValidationMessages($failures);
+
+    $storage = $form_state->getStorage();
+    $storage[self::VALIDATION_RESULT] = $validation_feedback;
+    $form_state->setStorage($storage);
+
+    $submit_form = TRUE;
+
+    foreach ($validation_feedback as $feedback_item) {
+      if ($feedback_item['status'] == 'todo' || $feedback_item['status'] == 'fail') {
+        $submit_form = FALSE;
+
+        break;
+      }
+    }
+
+    if ($submit_form === FALSE) {
+      $this->service_Messenger
+        ->addError('Your file import was not successful. Please check the Validation Result Window for errors and try again.');
+
+      $form_state->setRebuild(TRUE);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * DO NOT MODIFY.
+   */
+  public function form($form, &$form_state) {
+
+    $form = parent::form($form, $form_state);
+    $this->service_Messenger->addWarning('THIS IMPORTER IS FOR TESTING PURPOSES ONLY.');
+
+    $storage = $form_state->getStorage();
+    if (isset($storage[self::VALIDATION_RESULT])) {
+      $validation_result = $storage[self::VALIDATION_RESULT];
+
+      $form['validation_result'] = [
+        '#type' => 'inline_template',
+        '#theme' => 'validation_result_window',
+        '#data' => [
+          'validation_result' => $validation_result,
+        ],
+        '#weight' => -100,
+      ];
+    }
+
+    $form['file']['file_upload_existing']['#access'] = FALSE;
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * DO NOT MODIFY.
+   */
+  public function formSubmit($form, &$form_state) {
+
+    // Display successful message to user if file import was without any error.
+    $this->service_Messenger
+      ->addStatus('<b>Your file import was successful and a Job Process Request has been created to securely save your data.</b>');
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * DO NOT MODIFY.
+   */
+  public function run() {}
+
+  /**
+   * {@inheritdoc}
+   *
+   * DO NOT MODIFY.
+   */
+  public function postRun() {}
+
+  /**
+   * Valid delimited file process message.
+   *
+   * REMOVE IF NOT REQUIRED.
```

Comment:
Is this only here because we haven't yet finished with the process messages migration for this validator? If so, lets add a todo here to remind us to remove this in that PR.